src/main/java/de/tk/sys/security/cms/impl/PKCS7EncryptionServiceImpl.java

```diff
@@ -217,7 +216,7 @@ private Recipient determineKeyTransRecipient(
 	) throws IOException
 	{
 		Recipient recipient;
-		DERSequence parameters = (DERSequence) recipientFromContent.getKeyEncryptionAlgorithm().getParameters();
+		ASN1Sequence parameters = (ASN1Sequence) recipientFromContent.getKeyEncryptionAlgorithm().getParameters();

```

Comment:
not related to Google Guava or commons-io but makes sense anyway.

Reply:
True, thanks for merging!